frontend/src/utils/tsxUtils.tsx

```diff
@@ -385,45 +394,53 @@ export class StatusIndicator extends Component<StatusIndicatorProps> {
 
     componentWillUnmount() {
         clearInterval(this.timerHandle);
-        this.hide?.call(this);
-        this.hide = undefined;
+        this.toastRef && toast.close(this.toastRef)
+        this.toastRef = null
     }
 
     customRender() {
         const content =
-            <ChakraProvider theme={redpandaTheme}>
-                <Box mb="0.2em">
-                    <Box minW={300}>
-                        <Progress
-                            value={this.props.fillFactor * 100}
-                            isIndeterminate={this.props.statusText === 'Connecting'}
-                            colorScheme="brand"
-                        />
-                    </Box>
+            <Box mb="0.2em">
+                <Box minW={300}>
+                    <Progress
+                        value={this.props.fillFactor * 100}
+                        isIndeterminate={this.props.statusText === 'Connecting'}
+                        colorScheme="blue"
```

Comment:
[question] Here color scheme is blue, but before it was brand, is that intentional?

Reply:
@malinskibeniamin for now we only have the following variants for a Toast.
![Screenshot 2023-09-25 at 12 54 00](https://github.com/redpanda-data/console/assets/1083817/0e09efd9-787a-4d9c-8ab6-af5cd030a7d7)

So I had to change the progress to match the (blue) neutral colour. We can later come up with some neutral (gray) variant in the future? But I think it's worth revisiting once we have a styleguide? WDYT? 

![Screenshot 2023-09-25 at 12 55 57](https://github.com/redpanda-data/console/assets/1083817/aa27d2d0-5e33-4afd-ba5f-9601e30b5a4c)

Reply:
I will leave this one for @rikimaru0345 and @frankieshakes to decide